src/Symfony/Component/DependencyInjection/Tests/Compiler/IntegrationTest.php

```diff
@@ -234,6 +238,54 @@ public function getYamlCompileTests()
             $container,
         ];
     }
+
+    public function testTaggedServiceWithIndexAttribute()
+    {
+        $container = new ContainerBuilder();
+        $container->register(BarTagClass::class, BarTagClass::class)
```

Comment:
you don't need to repeat class and id twice when they are the same

Reply:
Right, forget that.
I'll correct that too.

Thanks